data/libraries.json

```diff
@@ -395,7 +395,8 @@
         "platforms": [
             "Go"
         ],
-        "url": "https://mellium.im/code"
+        "url": "https://mellium.im/code",
+        "logo": "https://mellium.im/mellium_icon.png"
```

Comment:
I couldn't get the site working to tell if this is actually working, but it does build so that's a good sign?